Documentation/Getting-Started-For-Windows.md

```diff
@@ -15,6 +15,7 @@ may be replaced in all commands below with the directory of your choice.
 * Clone and build CoreCLR:
     ```
     $ git clone https://github.com/dotnet/coreclr
+    $ git clone https://github.com/dotnet/corefx
```

Comment:
AFAIK there is no requirement to build corefx when building coreclr.

Nuget saw issues when there was an Azure network issue.